### PR TITLE
DW/settings dialog/WIP

### PR DESCRIPTION
**Pull request by RogodaThallus**
Adds two buttons for testing the default commit params and settings. Both are temporary until I can plug the settings into the drop-down bar and put the default dialog into its settings button.

Settings buttons are non functional currently. 
The default parameter dialog is functional. It is a copy of the commit dialog that is missing the call to commit.

### CODE REVIEW

File: jovian/jovian_nb_ext/main.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ define([
   "base/js/namespace",
   "base/js/dialog",
   "base/js/keyboard"
-], function($, Jupyter, dialog, keyboard) {
+], function ($, Jupyter, dialog, keyboard) {
```
Comment (PrajwalPrashanth) [inline jovian/jovian_nb_ext/main.js]: Install `Prettier - code formatter` vscode extension. to avoid these changes

Comment (PrajwalPrashanth) [reply jovian/jovian_nb_ext/main.js]: Hey @RogodaThallus install this extension for the next commits. These are not code changes, we want to keep the format consistent.

Comment (RogodaThallus) [reply jovian/jovian_nb_ext/main.js]: It is installed and was used prior to last commit. It should be fine now.

File: jovian/jovian_nb_ext/main.js [hunk 2]
```diff
@@ -609,6 +710,155 @@ define([
...
+            }
+          }
+        },
+        open: function() {
+          getParams().then(jvn_params => {
+            if (jvn_params == null) {
+              $("#nb_filename_box").val(
+                Jupyter.notebook.notebook_name.replace(".ipynb", "")
+              );
+              $($("input[name=secret_opt")[1]).prop("checked", true);
```
Comment (PrajwalPrashanth) [inline jovian/jovian_nb_ext/main.js]: These changes were merged and changed. Update the branch and resolve conflicts before next commit

Comment (PrajwalPrashanth) [reply jovian/jovian_nb_ext/main.js]: Pull the latest master and merge, you might have older master branch when you merged.

**Comment by RogodaThallus** [comment]
Notebook is done. The buttons work as intended.
image

**Comment by PrajwalPrashanth** [comment]
Im closing this PR as this was added to a another PR.

**Pull request closed without merging**